sdrf-proteomics/README.adoc

```diff
@@ -24,28 +24,87 @@ endif::[]
+* Use case guidelines: These are the guidelines for specific proteomics experiments (e.g inmunopeptidomics). Each use case has its own directory containing:
+- A detailed specification document of the Use case guidelines; including links to examples in GitHub.
+- A set of templates for the most common experiments for the use case.
+* Metadata guidelines: These are the guidelines for specific metadata annotations (e.g. patient pre-existing condition).
+
+image::../images/sdrf-guidelines-structure.png[Logo]
+
+**Figure 2**: SDRF-Proteomics specification structure. The main specification defines the core rules and is extended by specific use case guidelines and sample annotation guidelines. Each guideline type has associated templates.
+
+NOTE: The main specification is in the `sdrf-proteomics` directory. The use case guidelines are in the `use-cases` subdirectory within the `sdrf-proteomics` directory. Use case guidelines are extensions of the core specification, and should follow all the rules and requirements in the main specification. If a use case rule is in conflict with the specification, a note should be done in the main specification to reflect the extension or conflict. 
+
+== SDRF-Proteomics specification versioning
+
+The SDRF-Proteomics specification is versioned using the Semantic Versioning 2.0.0 (https://semver.org/) scheme. The version number is in the format MAJOR.MINOR.PATCH, where:
+
+* MAJOR version is incremented for incompatible changes to the specification, when major changes are done to the specification.
+
+* MINOR version is incremented for new features that are backward compatible with the previous version. Guidelines and templates are added or modified.
+
+* PATCH version is incremented for bug fixes and minor changes that do not affect the specification or the templates. This includes typos, formatting changes, and other minor updates.
+
+Every change in the specification should be done in GitHub using pull requests into the dev branch. The pull request should include a description of the changes and the reason for the changes. The pull request will be reviewed by the community and merged into the main branch when approved. After the merge, the version number will be updated according to the changes made, the release will be performed, and the Zenodo record will be updated.
+
+=== Adding the specification version to the file
+
+From version 1.1.0, the specification version MUST be added to each file as a column after factor values. The prefix of the column MUST be `sdrf-format`. The column name MUST be sdrf format version. The value of the column MUST be the version of the specification that was used to create the file.
```

Comment:
Do we need to add sdrf-format[sdrf format version] to template and examples? sdrf-format[sdrf format version] is mandatory from 1.1.0